packages/vike-react/renderer/onRenderClient.tsx

```diff
@@ -18,16 +19,18 @@ const onRenderClient: OnRenderClientAsync = async (pageContext): ReturnType<OnRe
     if (!root) {
       // First rendering
       root = ReactDOM.createRoot(container)
-    } else {
```

Comment:
I'm not sure this is correct to remove `else` here. On first rendering we don't need to do this as `title` and `lang` have already been properly set in the HTML by `onRenderHtml()`

Reply:
Sure, my bad 😁